File: src/Model/Timeslots.java
package Model;

public class Timeslots {

	private String startTime;
	private String endTime;

	public Timeslots(String startTime, String endTime) {

		this.startTime = startTime;
		this.endTime = endTime;
	}

	public String getStartTime() {
		return startTime;
	}

	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}

	public String getEndTime() {
		return endTime;
	}

	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}

	public boolean equals(Timeslots aTimeslot) {
		return this.startTime.equals(aTimeslot.startTime) &&
				this.endTime.equals(aTimeslot.endTime);
	}

}

File: src/Model/Timeslots.py
from sqlalchemy import Column, Integer, String
from database import Base  # Import Base from your database setup file

class Timeslot(Base):
    __tablename__ = 'timeslots'  # The name of the table in the database

    # Columns
    id = Column(Integer, primary_key=True, autoincrement=True)  # Primary key, auto-incremented
    start_time = Column(String, nullable=False)  # String to store the start time
    end_time = Column(String, nullable=False)  # String to store the end time

    def __init__(self, start_time, end_time):
        self.start_time = start_time
        self.end_time = end_time

    def __repr__(self):
        return f"Timeslot(start_time={self.start_time}, end_time={self.end_time})"

    def __eq__(self, other):
        if isinstance(other, Timeslot):
            return self.start_time == other.start_time and self.end_time == other.end_time
        return False
